audio-worklet/basic/js/noise-generator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+registerProcessor('noise-generator', class Noise extends AudioWorkletProcessor {
+  static get parameterDescriptors () {
+    return [{
+      name: 'gain',
+      defaultValue: 0.25,
+      minValue: 0,
+      maxValue: 1
```

Comment:
For a general noise generator, I would think maxValue would be unlimited, more or less. For this demo, I guess it's ok; you're not trying to make this a general purpose noise generator.